pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/queue/EventDao.java

```diff
@@ -5,51 +5,90 @@
+            + "FROM EVENT_QUEUE " + "WHERE event_type = ? and ddp_participant_id = ?  ";
+
+    private static String SQL_BY_TRIGGER_TRUE_CONDITION = "AND event_triggered = 1";
+
+    private static String GET_EVENT_BY_EVENT_TYPE_AND_KIT_REQUEST_ID = "SELECT "
+            + "event_id, event_date_created, event_type, ddp_instance_id, dsm_kit_request_id, ddp_participant_id, event_triggered "
+            + "FROM EVENT_QUEUE " + "WHERE event_type = ? AND dsm_kit_request_id = ?";
+
+    private static final String SQL_INSERT_EVENT =
+            "INSERT INTO EVENT_QUEUE SET event_date_created = ?, event_type = ?, ddp_instance_id = ?, "
+                    + "ddp_participant_id = ?, dsm_kit_request_id = ?, event_triggered = ?";
+
+    public static final String SQL_SELECT_KIT_FOR_REMINDER_EMAILS =
+            "SELECT eve.event_name, eve.event_type, request.ddp_participant_id, request.dsm_kit_request_id, realm.instance_name, "
+                    + "realm.base_url, "
+                    + "realm.ddp_instance_id, realm.auth0_token, realm.notification_recipients, kit.receive_date, kit.scan_date, "
+                    + "request.upload_reason, request.ddp_kit_request_id, "
+                    + "(SELECT count(role.name) FROM ddp_instance realm2, ddp_instance_role inRol, instance_role role "
+                    + "WHERE realm2.ddp_instance_id = inRol.ddp_instance_id AND inRol.instance_role_id = role.instance_role_id "
+                    + "AND role.name = ? AND realm2.ddp_instance_id = realm.ddp_instance_id) AS 'has_role' "
+                    + "FROM ddp_kit_request request LEFT JOIN ddp_kit kit ON (kit.dsm_kit_request_id = request.dsm_kit_request_id) "
+                    + "LEFT JOIN ddp_instance realm ON (request.ddp_instance_id = realm.ddp_instance_id) "
+                    + "LEFT JOIN ddp_participant_exit ex ON (ex.ddp_participant_id = request.ddp_participant_id "
+                    + "AND ex.ddp_instance_id = request.ddp_instance_id) "
+                    + "LEFT JOIN event_type eve ON (eve.ddp_instance_id = request.ddp_instance_id "
+                    + "AND eve.kit_type_id = request.kit_type_id "
+                    + "AND eve.event_type = 'REMINDER') "
+                    + "LEFT JOIN EVENT_QUEUE queue ON (queue.DSM_KIT_REQUEST_ID = request.dsm_kit_request_id "
+                    + "AND queue.EVENT_TYPE = eve.event_name) "
+                    + "WHERE ex.ddp_participant_exit_id IS NULL AND kit.scan_date IS NOT NULL "
+                    + "AND kit.scan_date <= (UNIX_TIMESTAMP(NOW())-(eve.hours*60*60))*1000 "
+                    + "AND kit.receive_date IS NULL AND kit.deactivated_date IS NULL AND realm.is_active = 1 AND queue.EVENT_TYPE IS NULL";
+
 
     @Override
     public int create(EventDto eventDto) {
-        return 0;
+        throw new NotImplementedException("Not implemented, use insertEvent method instead");
```

Comment:
Thanks for doing this, making it explicit to callers that this method is busted.  Although it would be nice to have consistent implementors of `Dao`, this small change here is helpful.